src/textual/widgets/_progress_bar.py

```diff
@@ -353,6 +423,8 @@ def updater(percentage: float | None) -> None:
         with Horizontal():
             if self.show_bar:
                 bar = Bar(id="bar")
+                bar.colorscheme = self._colorscheme
+                bar.thickness = self._thickness
```

Comment:
What do you think of extending `Bar.__init__` to also accept these two values as arguments?

Reply:
Fixed.